.github/workflows/build_wheel.yml

```diff
@@ -0,0 +1,281 @@
+name: Build Wheels
+
+on:
+  push:
+    branches:
+      - main
+    tags:
+      - "v*"
```

Comment:
It's just a convention, but since there are multiple dtypes in the repo, I think it's less confusing to only look for tags like `quaddtype-v*`.